### PR TITLE
[ENH] Make png default

### PR DESCRIPTION
**Pull request by jbteves**
Closes None

Changes proposed in this pull request:

- Makes png generation default
- Adds `--nopng`
- Adds check to override `--nopng` if `--png-cmap` is supplied

### CODE REVIEW

File: tedana/workflows/tedana.py [hunk 1]
```diff
@@ -261,11 +262,11 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         which is slightly slower but may be more accurate.
     verbose : :obj:`bool`, optional
         Generate intermediate and additional files. Default is False.
-    png : obj:'bool', optional
-        Generate simple plots and figures. Default is false.
+    nopng : obj:'bool', optional
```
Comment (emdupre) [inline tedana/workflows/tedana.py]: I'm not sure it's worth exposing this -- it'd make sense to me to generate figures by default. WDYT ?

**Comment by jbteves** [comment]
@emdupre Can you give this a look?

File: tedana/workflows/tedana.py (hunk 1 above)
Comment (jbteves) [reply tedana/workflows/tedana.py]: I think this is fixed in f6cba2e

**Comment by emdupre** [comment]
Total nit-pick: could we change `nopng` to `no_png` ? :smile:

**Pull request merged**